gwt/pom.xml

```diff
@@ -40,39 +40,10 @@
       <classifier>sources</classifier>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <classifier>sources</classifier>
```

Comment:
Can you explain what this does? It doesn't unpack the sources? Does it tell the consumer of dagger-gwt that they should include javax.inject sources instead of .class files?